doc/whatsnew/2.4.rst

```diff
@@ -220,3 +220,10 @@ When enabling/disabling several messages and groups in a config file,
 it can be unclear which messages are actually enabled and which are disabled.
 This new command produces the final resolved lists of enabled/disabled messages,
 sorted by symbol but with the ID provided for use with ``--help-msg``.
+
+
+* Add new good-names-rgx and bad-names-rgx to enable white-/blacklisting of regular expressions
```

Comment:
Can you move this to 2.5 instead? It's going to be released as part of 2.5.

Reply:
done